modules/tensor_mechanics/src/kernels/StressDivergenceRSphericalTensors.C

```diff
@@ -27,19 +27,13 @@ StressDivergenceRSphericalTensors::StressDivergenceRSphericalTensors(const Input
 Real
 StressDivergenceRSphericalTensors::computeQpResidual()
 {
-  Real div = 0;
+  mooseAssert(_assembly.coordSystem() == Moose::COORD_RSPHERICAL,
```

Comment:
Is this really necessary to do in `computeQpResidual()`?  Do we possibly have a `subdomainSetup()` or something we can use instead?  In `dbg` mode this is a bummer (but not a HUGE) bummer.